src/main/java/frc/robot/RobotContainer.java

```diff
@@ -61,19 +69,128 @@ public class RobotContainer {
   SwerveInputStream driveAngularVelocitySim =
       SwerveInputStream.of(
               drivebase.getSwerveDrive(), () -> driverXbox.getLeftY(), () -> driverXbox.getLeftX())
-          .withControllerRotationAxis(() -> driverXbox.getRawAxis(2))
+          .withControllerRotationAxis(() -> driverXbox.getRawAxis(4))
           .deadband(OperatorConstants.DEADBAND)
           .scaleTranslation(1)
           .allianceRelativeControl(true);
 
   Command driveFieldOrientedAngularVelocitySim =
       drivebase.driveFieldOriented(driveAngularVelocitySim);
 
+  SwerveInputStream driveRobotOriented =
+      driveAngularVelocitySim.copy().robotRelative(true).allianceRelativeControl(false);
+  Command driveRobotOrientedAngularVelocity = drivebase.driveFieldOriented(driveRobotOriented);
```

Comment:
```suggestion
  private final SwerveInputStream driveRobotOriented =
      driveAngularVelocitySim.copy().robotRelative(true).allianceRelativeControl(false);
  private final Command driveRobotOrientedAngularVelocity = drivebase.driveFieldOriented(driveRobotOriented);
```

There are some other variables in here that should be `private final` too